CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 - Fixed the SIGPIPE signal handler so Firecracker no longer exits. The signal
   is still recorded in metrics and logs.
+- Snapshot related host files (vm-state, memory, block backing files) are now
+  flushed to their backing mediums as part of the CreateSnapshot operation.
```

Comment:
Do we need a knob here ? There are also usecases when it is safe to allow the pagecache to deal with this.

Reply:
I am not sure the added knob complexity (and especially understanding when to use it correctly) is worth the gains. Is CreateSnapshot an operation that is time-critical/time-sensitive enough to start messing with knobs that can break consistency when misused?

Conceptually I see the `CreateSnapshot` operation as a way to snapshot a live VM to a persistent, stable medium. If users need this operation to be fast, they can just use a fast medium (like nvme or ramfs, tmpfs, etc) without needing tricky knobs in `firecracker`.